src/pages/BrowserPage.qml

```diff
@@ -310,6 +312,9 @@ Page {
     Connections {
         target: WebUtils
         onOpenUrlRequested: {
+            // Refuse if blocked by MDM
+            if (!AccessPolicy.browserEnabled) return
```

Comment:
I'd still bring browser to the foreground.

```
if (!AccessPolicy.browserEnabled) {
    bringToForeground(webView.chromeWindow)
    window.activate()
    return
}
```

Reply:
Done